po/es.po

```diff
@@ -5917,15 +5939,15 @@ msgstr ""
 "Pulse el botón central sobre el preajuste para establecerlo en un módulo "
 "nuevo"
 
-#: ../src/develop/imageop.c:2807 ../src/develop/imageop.c:2833
+#: ../src/develop/imageop.c:2820 ../src/develop/imageop.c:2846
 msgid "ERROR"
 msgstr "ERROR"
 
-#: ../src/develop/imageop.c:3174
+#: ../src/develop/imageop.c:3187
 msgid "purpose"
-msgstr "finalidad"
+msgstr "finalidad\t"
```

Comment:
Is this tab character intentional here? (to correct for imperfect outlining in the module tooltip?) That will not necessarily work the same way across fonts and other css changes.

Reply:
> Is this tab character intentional here? (to correct for imperfect outlining in the module tooltip?) 

Yes, indeed. I figured out there is misalignment text maybe due to differ the length of the words translated. These images show that:
Without tab
![photo_2021-01-13_12-30-24](https://user-images.githubusercontent.com/47407674/104498073-23157000-55d3-11eb-8cb8-a6ec1a488e08.jpg)
With spaces
![photo_2021-01-13_13-20-18](https://user-images.githubusercontent.com/47407674/104507263-632e2000-55de-11eb-9a8c-3a31471d14e4.jpg)
**With tab**
![photo_2021-01-13_12-30-38](https://user-images.githubusercontent.com/47407674/104498070-227cd980-55d3-11eb-83f9-334b98a007b3.jpg)

> That will not necessarily work the same way across fonts and other css changes.

I think so. But I also think that with that change, at least it is much more readable with the defaults and it only has that effect to adapt it to the text translated. But if there is any idea or proposal for a better solution, I will adapt it.

Reply:
What I mean is, it may be better on _your_ system with this adjustment, but on _my_ system, with different fonts, it is worse :-)

BTW @TurboGit I tried to improve on the outlining algorithm by measuring the width of pango layouts. But the problem (I think) I have is that I can't access the correct pango context for the tooltip. Aligning tabs in tooltips is difficult (maybe impossible). I fought with that for the history tooltips. The only way I found to make it work is to use monospaced. Which is doable in this case also (just for the purpose/input/output words) by setting a markup text for the tooltip and using <tt></tt>. But there's a bit of extra hassle involved because the remaining text needs to be escaped to not confuse the markup processor and g_markup_printf_escaped seems to have a problem here (maybe it doesn't support the %.*s construct you use here to get the correct number of tabs.